public/locale/en.json

```diff
@@ -1616,7 +1616,7 @@
   "phone_number": "Phone Number",
   "phone_number_at_current_facility": "Phone Number of Contact person at current Facility",
   "phone_number_min_error": "Phone number must be at least 10 characters long",
-  "phone_number_must_be_10_digits": "Phone number must be a 10-digit mobile number",
+  "phone_number_required": "Phone number is required",
```

Comment:
like i said already, `field_required` is sufficient. this validator might be used in Fax and other purposes as well (as per FHIR)